rectools/metrics/__init__.py

```diff
@@ -72,4 +73,5 @@
     "PairwiseDistanceCalculator",
     "PairwiseHammingDistanceCalculator",
     "SparsePairwiseHammingDistanceCalculator",
+    "HitRate",
```

Comment:
It's better to put this metric close to other classification metrics, after the `Accuracy` e.g.

Reply:
Moved up to classification ones